cypress/integration/table/search.spec.ts

```diff
@@ -40,21 +40,23 @@ describe('Search', () => {
       }).as('getSettings');
 
       cy.visit('/', {
+        // For some reason this doesn't work with date pickers v6
+        // Commenting out for now
```

Comment:
This fix was originally a work around to ensure that desktop date pickers are used. If this fix is no longer required, then we can remove them